apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -340,7 +315,7 @@ export class SendMessagePush extends SendMessageBase {
           status: ExecutionDetailsStatusEnum.SUCCESS,
           isTest: false,
           isRetry: false,
-          raw: JSON.stringify(result),
+          raw: JSON.stringify({ result, deviceToken }),
```

Comment:
We add the device token in the response, though it can be arguable if we shouldn't as identifies the device of a user. We need to be careful with them.

Some providers treat the device token as not PII, like for example OneSignal:
https://documentation.onesignal.com/docs/handling-personal-data#push-tokens, but we should analyse our own risks.
CC: @scopsy @Cliftonz @djabarovgeorge @jainpawan21

Reply:
@scopsy is it ok to proceed?